drivers/hmis/app/graphql/types/hmis_schema/unit.rb

```diff
@@ -62,5 +63,12 @@ def name
     def latest_opportunity
       load_ar_association(object, :latest_opportunity)
     end
+
+    def accepting_ce_referrals
+      latest_opportunity = load_ar_association(object, :latest_opportunity)
+      return false if latest_opportunity.nil?
+
+      load_ar_association(latest_opportunity, :referrals, scope: Hmis::Ce::Referral.active).empty?
```

Comment:
Future improvement might be to instead add an association like `has_many :active_referrals` on `Unit` (through opportunity), if possible. That would make debugging easier (can call `unit.active_referrals` directly) and simplify the data loading.